docs/02-app/02-api-reference/05-next-config-js/windowHistorySupport.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: webVitalsAttribution
+description: Learn how to use the webVitalsAttribution option to pinpoint the source of Web Vitals issues.
```

Comment:
```suggestion
title: windowHistorySupport
description: Learn how to use the windowHistorySupport to sync the History API with the Next.js Router. 
```

Reply:
You probably forgot to commit this change

I made a follow up #60503

Reply:
Thank you 🙏🏼